src/waypoint.jsx

```diff
@@ -107,6 +108,10 @@ export default class Waypoint extends React.Component {
     }
   }
 
+  componentWillReceiveNewProps(newProps) {
```

Comment:
The change from "componentWillReceiveProps" to "componentWillReceiveNewProps" seems like a mistake. I'm pretty sure the latter isn't a lifecycle method in React.

Reply:
Oops, yes, totally a mistake! I'm not even sure how this happened 😮